tiscale_connector.py

```diff
@@ -184,7 +183,7 @@ def _parse_report_status_msg(self, response, action_result, data):
         # pprint.pprint(response)
         response1['tiscale_link'] = "{0}{1}".format(
             self._base_url, "?q=" + data['hash'])
-        print "TISCALE link:" + response1['tiscale_link']
+        print("TISCALE link:" + str(response1['tiscale_link']))
```

Comment:
can we either make this a `debug_print` or remove it? also, use string formatting instead of concatenation if it's kept